query_postal_codes_heatmaps_api.py

```diff
@@ -0,0 +1,98 @@
+import random
+import statistics
+from collections import defaultdict
+
+import matplotlib.pyplot as plt
+from datetime import datetime, timedelta, date
```

Comment:
`datetime` is part of the Python standard library, so you should move it into the 1st block.

Reply:
Indeed. fixed